src/graphql/data/schema.graphql

```diff
@@ -0,0 +1,398 @@
+"""This directive allows results to be deferred during execution"""
```

Comment:
confirming you want to check this in?

Reply:
Yes following connors thoughts, seems like it's best to keep the schema in the repo so the build doesn't have to fetch every time